frontend/src/app/components/auth/register/register.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import { Validators, FormBuilder } from '@angular/forms';
+import { AuthService } from '../../../services/auth.service';
+import { User } from '../../../model/user';
+
+@Component({
+  selector: 'bm-register',
+  templateUrl: './register.component.html',
+  styleUrls: ['./register.component.scss']
+})
+export class RegisterComponent implements OnInit {
+  user: User;
+  registerForm = this.fb.group({
+    registerFormEmail: ['', [Validators.required, Validators.email]],
+    registerFormPassword: ['', [Validators.required, Validators.minLength(6)]],
+    registerFormPasswordConfirmation: ['', Validators.required]
```

Comment:
You have to create custom validator for password confirmation
```
passwordMatchValidator(fg: FormGroup){
  return fg.get('registerFormPassword').value === fg.get('registerFormPasswordConfirmation').value ? null : { 'mismatch' : true};
}
```

Reply:
ok